tap-agent/src/agent/sender_allocation.rs

```diff
@@ -352,7 +366,7 @@ impl SenderAllocationState {
         })
     }
 
-    async fn initialize_unaggregated_receipts(&self) -> Result<UnaggregatedReceipts> {
+    async fn refresh_all_database(&self) -> Result<UnaggregatedReceipts> {
```

Comment:
I'm not a massive fan of this name. Is this only used to calculate unaggregated fees? If so, could we call it `calculate_unaggregated_fees`?

Reply:
We have two functions that recalculate unaggregated fees.

One recalculates up to the `last_id` in our state, and the other recalculates the whole database (where last_id < max::u64).

Reply:
I just didn't want to have a `calculate_unaggregated_fee_for_whole_database()`, do you have any suggestion that can differentiate them better?

Reply:
done